src/client/discover/DiscoverUser.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import _ from 'lodash';
+import { attempt, isError, has, get } from 'lodash';
 import { Link } from 'react-router-dom';
 import { FormattedMessage } from 'react-intl';
 import Avatar from '../components/Avatar';
 import FollowButton from '../widgets/FollowButton';
 import WeightTag from '../../client/components/WeightTag';
 
 const DiscoverUser = ({ user, isReblogged }) => {
-  const parsedJSON = _.attempt(JSON.parse, user.json_metadata);
-  const userJSON = _.isError(parsedJSON) ? {} : parsedJSON;
-  const userProfile = _.has(userJSON, 'profile') ? userJSON.profile : {};
-  const name = userProfile.name;
+  const parsedJSON = attempt(JSON.parse, user.json_metadata);
+  const userJSON = isError(parsedJSON) ? {} : parsedJSON;
+  const profileName = get(userJSON, 'profile.name');
```

Comment:
I'd recommend to use other notation to get property: get(userJSON, ['profile', 'name']);